skills/easter-eggs.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * When the bot hears a specific phrase, show an easter egg
+ */
+
+module.exports = (controller) => {
+
+    // When someone asks what does the fox say, show a gif!
+
+    const whatDoesTheFoxSayString = [
```

Comment:
Listen to any mention for `what does the fox say?` with or without the question mark.